File: 01_Language/01_Functions/java/src/main/java/com/onevgo/functions/ArrayValues.java
package com.onevgo.functions;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;
import java.util.Map;

public class ArrayValues {
    public static <K, V> List<V> arrayValues(Map<K, V> map) {
        return new ArrayList<>(map.values());
    }

    public static void main(String[] args) {
        HashMap<String, String> map = new HashMap<>();
        map.put("size", "XL");
        map.put("color", "gold");
        System.out.println(arrayValues(map));
    }
}

File: 01_Language/01_Functions/js/array_values.js
function array_values(arr) {
    if (Array.isArray(arr)) {
        return arr.slice(0);
    }
    let res = []
    let j = 0
    for (let i in arr) {
        res[j++] = arr[i]
    }
    return res;
}

let array = {"size": "XL", "color": "gold"}
console.log(array_values(array));
